docs/building.md

```diff
@@ -64,6 +64,7 @@ See the [documentation on testing](../tools/slang-test/README.md) for more infor
 | `SLANG_EMBED_STDLIB_SOURCE`       | `TRUE`           | Embed stdlib source in the binary                                  |
 | `SLANG_ENABLE_ASAN`               | `FALSE`          | Enable ASAN (address sanitizer)                                    |
 | `SLANG_ENABLE_FULL_IR_VALIDATION` | `FALSE`          | Enable full IR validation (SLOW!)                                  |
+| `SLANG_ENABLE_IR_BREAK_ALLOC`     | `FALSE`          | SLANG_ENABLE_IR_BREAK_ALLOC                                        |
```

Comment:
I am not sure why this change is a part of this PR.
But the 3rd column should have an explanation of what the macro is rather than just repeating the macro keyword

Reply:
Done.

Reply:
I needed this for debugging, somehow this is no longer enabled by default when in debug build after we switched to cmake.